src/templates/next/layouts/Skeleton/Skeleton.tsx

```diff
@@ -19,55 +19,40 @@ export const Skeleton = ({
     timeNow.getFullYear()
 
   return (
-    <html lang={page.language || "en"} data-theme={`isomer-${site.theme}`}>
-      <head>
-        <meta charSet="utf-8" />
-        <meta
-          name="viewport"
-          content="width=device-width, user-scalable=yes, initial-scale=1.0"
-        />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-
-        <meta property="og:title" content={page.title} />
-        {page.noIndex && <meta name="robots" content="noindex" />}
-
```

Comment:
Yup, we cannot nest the `<html>` tag inside a React component (will throw a bunch of errors), so we need to rely on another method to inject them.

For `<head>` tag as well, the contents inside the tag needs to be set in a specific way for Next.js, otherwise the framework will override the contents set inside (especially the `<title>` tag).